src/components/Confirm/index.js

```diff
@@ -9,24 +9,27 @@ import { preventDefaultOnClick } from '~utils/index'
 import { uploadDocument, uploadLivePhoto, uploadLiveVideo } from '~utils/onfidoApi'
 import CaptureViewer from './CaptureViewer'
 import { poaDocumentTypes } from '../DocumentSelector/documentTypes'
+import Button from '../Button'
 import Error from '../Error'
 import Spinner from '../Spinner'
 import Title from '../Title'
 import { trackException, trackComponentAndMode, appendToTracking, sendEvent } from '../../Tracker'
 import { localised } from '../../locales'
 
 const RetakeAction = localised(({retakeAction, translate}) =>
-  <button onClick={retakeAction}
-    className={`${theme.btn} ${theme['btn-outline']} ${style.retake}`}>
+  <Button
+    onClick={retakeAction}
+    className={style.retake}
+    variants={["outline"]}
+  >
     {translate('confirm.redo')}
-  </button>
+  </Button>
 )
 
 const ConfirmAction = localised(({confirmAction, translate, error}) =>
-  <button href='#' className={`${theme.btn} ${theme["btn-primary"]}`}
```

Comment:
Removing this class `theme["btn-primary"]` will make the tests fail because cucumber will be looking for this class in order to proceed to the next screen

Reply:
Ahh, I missed that. I've fixed up the E2E test references so that they refer to the new expected class name :)